compiler/src/parsing/well_formedness.re

```diff
@@ -51,6 +52,8 @@ let prepare_error =
         )
       | NoLetRecMut(loc) =>
         errorf(~loc, "let rec may not be used with the `mut` keyword.")
+      | NumberDoesntFit(num, loc) =>
+        errorf(~loc, "Grain does not support numbers this big: %s", num)
```

Comment:
I thought mine was friendlier...

Reply:
I liked yours! The main problem was the `Grain does not support ...` because it mentions Grain in a negative way. When you read "Grain doesn't do X" your brain goes straight to "find you a language that will." 😂

I joke, but I'm completely serious.

I do think that the "this big" part was very friendly, but it begs the question "well how big can they be?", which means the user has to go hunt down the documentation (which we don't have (which further pushes the negative feeling)) when we can just inline the docs. I would definitely like to emulate the friendliness of how you originally did it, but I wasn't really sure about how to do that while still giving the technical details.

Reply:
Maybe something like `Numbers larger than 2^63 or smaller than -2^63 + 1 are unsupported.` could work better. @belph and @phated what do you think?